production/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# syntax=docker/dockerfile:1
+
+FROM node:16-bullseye-slim
+
+RUN apt update && apt install -y ca-certificates
+
+ENV NODE_ENV=production
+WORKDIR /app
+
+COPY ["package.json", "./"]
+RUN npm install --production
+
+ARG TEMPORAL_SERVER="host.docker.internal:7233"
```

Comment:
Why put these in args?
It's typically something you'd provide at container run time, not bake into the image since this image could be used in different environments.